reference/config_files/client_certificates.rst

```diff
@@ -0,0 +1,24 @@
+.. _reference_config_files_client_certificates:
+
+
+Client certificates
```

Comment:
This is not a first level item. It is just 1 config item from ``global.conf``. It is not at the same level as settings, profiles, etc.